evap/staff/templates/quick-review.js

```diff
@@ -111,6 +113,35 @@ $(document).ready(() => {
         }
     }
 
+    var nextEvaluationIndex = 0;
+    setNextEvaluation(0);
+
+    $("#skip-next-evaluation").on("click", event => {
+        const skippedEvaluationId = $("[data-evaluation").not(":hidden").data("evaluation");
```

Comment:
```suggestion
        const skippedEvaluationId = $("[data-evaluation]").not(":hidden").data("evaluation");
```
Unless the selector without `]` has a special meaning unknown to me